vllm/utils.py

```diff
@@ -107,7 +107,7 @@
     "currently not supported for encoder/decoder "
     "models.")
 
-STR_NOT_IMPL_ENC_DEC_LORA = ("LoRA is currently not currently "
+STR_NOT_IMPL_ENC_DEC_LORA = ("LoRA is not currently "
```

Comment:
```suggestion
STR_NOT_IMPL_ENC_DEC_LORA = ("LoRA is currently not "
```

To be consistent with the other error messages

Reply:
Oh, in that case let's just keep it. I saw that it's the other way round for the messages above this one.